data/types.txt

```diff
@@ -1363,6 +1363,7 @@ amenity|recycling|container
 *railway|station|subway|bangkok
 *railway|station|subway|beijing
 *railway|station|subway|bengalore
+*railway|station|subway|bilbao
```

Comment:
I think new type shouldn't be added in the middle - number of line here is id of type.
Maps from previous release wouldn't be backward compatible

AFAIK it's ok to replace `mapswithme` or add at the end